src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1114,5 +1119,282 @@ public void SarifTransformerTests_ToCurrent_BasicResult()
 
             v2LogText.Should().Be(v2LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""files"": {
+                        ""http://localhost:34420/HtmlFixes.html"": {
+                          ""mimeType"": ""text/html""
+                        }
+                      },
+                      ""results"": [
+                        {
+                          ""ruleId"": ""WEB1079"",
+                          ""formattedRuleMessage"": {
+                            ""formatId"": ""default"",
+                            ""arguments"": [
+                              ""shape""
+                            ]
+                          },
+                          ""suppressionStates"": [ ""suppressedExternally"" ],
+                          ""baselineState"": ""existing"",
+                          ""level"": ""error"",
+                          ""snippet"": ""<area alt=\""Here is some text\"" coords=\""10 20 20\"" href=\""moon.html\"" shape=circle xweb:fixindex=\""0\"" />"",
+                          ""locations"": [
+                            {
+                              ""analysisTarget"": {
+                                ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                              },
+                              ""resultFile"": {
+                                ""uri"": ""http://localhost:34420/HtmlFixes.html"",
+                                ""region"": {
+                                  ""startLine"": 20,
+                                  ""startColumn"": 69,
+                                  ""endColumn"": 74,
+                                  ""offset"": 720,
+                                  ""length"": 5
+                                }
+                              }
+                            }
+                          ],
+                          ""fixes"": [
+                            {
+                              ""description"": ""Wrap attribute values in single quotes."",
+                              ""fileChanges"": [
+                                {
+                                  ""uri"": ""http://localhost:34420/HtmlFixes.html"",
+                                  ""replacements"": [
+                                    {
+                                      ""offset"": 623,
```

Comment:
For consistency: This should be 720., and the second offset should be 725.

Reply:
Fixed